### DISCUSSION TITLE
Need help for add option to load podman use netavark in rootless mode?

### DISCUSSION THREAD

**Discussion by calvin2021y** [containers/podman#20981] [Q&A]
Podman in rootless mode try use slirp4netns, I want force it use netavark since I has a way to workaround the root user limit.

Is there a options for this?

**Comment by calvin2021y** [comment]
I use podman inside podman with docker-compose, is there a way to add "--security-opt unmask=ALL" into dock-compose.yaml or put into global podman configure file?

**Comment by calvin2021y** [comment]
Is there a way to add unmask=ALL into `/etc/containers/containers.conf`, I can not find document

**Reply by rhatdan** [reply]
Sorry we don't support this at this time.  Open up an issue to add it, if you think it is valuable.
